NewForce-Cohort-9/tabloidmvc-stafford-infection-1
# Bb stafford view all categories

# Description

I have completed the ticket to view all categories. In doing so I have created a category management link in the navbar, so when the admin selects it, it directs them to the page with all of the categories, where they are able to see the list of all available categories in alphabetical order.

Trello Ticket: https://trello.com/c/58Eylq1E/9-9-view-all-categories


## Type of change: New Feature


# Testing Instructions

1. Clone down the repository
2. git checkout main
3. git fetch --all
4. git checkout bb-stafford
5. If you don't see the most recent version of my files, git pull origin bb-stafford
6. Open the project (TabloidMVC) in Visual Studio
7. If you haven't already, create a database called "TabloidMVC." The data we are using can be found under the "SQL" folder when looking at our group repository. Be sure to run a query for both files (Create_DB and Seed_Data)
8. Start the server. It should automatically open a webpage on your browser.
9. You'll be greeted by a page that will say "Blogs & Journals", "Get Started", and "Login." Click on Login
10. Login using [admin@example.com] and hit "Submit"
11. When logged locate the nav bar that should have "Home", "Posts", Tag Management", "Category Management" and "Logout".
12. Using the navbar at the top, navigate to "Category Management"
13. When clicked it will open to the Category Management page where the categories will be displayed in alphabetical order.
14. The list should be, in order, (Cooking, Cthulhu Sightings, Entertainment, History, Home and Garden, Improv, Movies, Music, Politics, Regrets, Science, Technology).
15. Now that all categories can be seen you can press the log out button.


# Checklist:

1. I have opened bb-stafford in my local repository

2. I have ran my application in a browser window to see the 'Category Management' on the navbar

3. After logging in the appropriate credentials, [admin@example.com], and clicking on 'Category
Management' in the navbar, it directs me to the page where you can view all of the available Categories in alphabetical order

4. I have added test instructions that can be followed to test this code, to view this new feature.
